src/components/smart-container/smart-container.js

```diff
@@ -36,7 +37,14 @@ const mapStateToProps = state => ({
  * </SmartContainer>
  * @extends {Component}
  */
-class SmartContainer extends Component {
+class SmartContainer extends BaseComponent {
+  /**
+   * @param {Object} obj an object
+   */
+  constructor(obj: Object) {
```

Comment:
I think it is better to keep this a regular component and pass in the player id

Reply:
changed.